content/docs/how-we-work/baseline/logging-details.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Logging
+weight: 2
```

Comment:
```suggestion
weight: 13
```

I know there are no 13 pages under this section but in case we add more sub-pages, I would like it to maintain the order we have for the headings on the baseline page. Setting the weight to match that order right now makes it easier for us later.